cli/src/cmd/forge/snapshot.rs

```diff
@@ -281,8 +277,8 @@ fn write_to_snapshot_file(
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub struct SnapshotDiff {
     pub signature: String,
-    pub source_gas_used: TestKindGas,
-    pub target_gas_used: TestKindGas,
+    pub source_gas_used: TestKindReport,
+    pub target_gas_used: TestKindReport,
```

Comment:
Same here as above - if this is going to be a report, we should change these names to reflect that